### PR TITLE
chore: update art.json doc

### PR DESCRIPTION
**Pull request by eugbyte**
Display modality for ART Certs

### CODE REVIEW

File: static/documents/pdt-v2-art-healthcert.json [hunk 1]
```diff
@@ -1,335 +1,334 @@
...
-                  "text": "287bf3a2-b2cc-4802-9c33-e601f692ae36:string:MacRitchie Hospital, Thomson Road, Singapore 123000"
-                }
-              }
-            ]
-          }
+        "notarisationMetadata": {
+            "reference": "9d22aacf-c3a5-492f-b717-a230c631e467:string:78073ba2-2a4a-4e62-9b09-a537d3c93f34",
+            "notarisedOn": "de3b68ff-c29a-44f9-845f-9da2b1b57e60:string:2022-03-11T01:58:09.723Z",
+            "passportNumber": "18539051-9b23-457a-ad48-e4dd0d544263:string:E7831177G",
+            "url": "21647938-88b4-4ec5-9119-43efd801414d:string:https://dev.verify.gov.sg/verify?q=%7B%22type%22%3A%22DOCUMENT%22%2C%22payload%22%3A%7B%22uri%22%3A%22http%3A%2F%2Flocalhost%3A5000%2Fstg%2Fdocument%2Fec4bcba7-5115-4349-af23-8fac4f53a9af%22%2C%22permittedActions%22%3A%5B%22VIEW%22%2C%22STORE%22%5D%2C%22redirect%22%3A%22https%3A%2F%2Fdev.verify.gov.sg%2Fverify%22%7D%7D#%7B%22key%22%3A%220304316b78888aa483fa1a34bdf445cf7dcbbf36eb8452194adf89b798bee6ad%22%7D"
```
Comment (HJunyuan) [inline static/documents/pdt-v2-art-healthcert.json:308]: Hey @eugbyte, I think this URL is not valid.

Perhaps I can help you update the ART sample in #77 as well?

Comment (eugbyte) [reply static/documents/pdt-v2-art-healthcert.json:308]: Hey thanks!

Comment (eugbyte) [reply static/documents/pdt-v2-art-healthcert.json:308]: so i close this PR?

Comment (HJunyuan) [reply static/documents/pdt-v2-art-healthcert.json:308]: Sure. Can help me approve the other PR too 🙏🏻

**Pull request closed without merging**